### PR TITLE
Proper description for the additional resources

### PR DESCRIPTION
**Pull request by marcusicaro**
Changed [This link] to [HTML Text Formatting] for better understanding of the content for visual impaired people.

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

**Comment by zachmmeyer** [comment]
Just to be sure: are you updating an existing link or are you adding a new one. Your title and description says you are updating, the when checking the the files changed it shows you are adding a brand new link.

**Comment by marcusicaro** [comment]
Hey, @zachmmeyer 

I'm adding a new one. I had a previous PR of this issue, (as shown here: https://github.com/TheOdinProject/curriculum/pull/24564#issuecomment-1264747079) but was suggested by @wise-king-sullyman to make a few changed. I've done those changed and then made a new PR.

Did I do right?

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
I think this reads a bit cleaner, what do you think?

File: foundations/html_css/html-foundations/working-with-text.md [hunk 1]
```diff
@@ -186,4 +186,4 @@ This section contains questions for you to check your understanding of this less
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
 *   [The semantic difference between &lt;strong> and &lt;b> or &lt;em> and &lt;i> tags and when to use them.](https://medium.com/@zac_heisey/when-to-use-strong-b-em-and-i-tags-in-your-markup-fa4d0af8affb)
-
+*   [HTML Text Formatting](https://www.w3schools.com/html/html_formatting.asp) article from W3Schools contains some great exercise to practice working with text in HTML.
```
Comment (wise-king-sullyman) [inline foundations/html_css/html-foundations/working-with-text.md:189]:
```suggestion
*   [An interactive HTML text formatting article](https://www.w3schools.com/html/html_formatting.asp)
```

Comment (marcusicaro) [reply foundations/html_css/html-foundations/working-with-text.md:189]: Hey, that's it! Way better @wise-king-sullyman

Comment (marcusicaro) [reply foundations/html_css/html-foundations/working-with-text.md:189]: Should I submit it again with the revision you proposed?

**Review by wise-king-sullyman — APPROVED** [review]
This looks great, thank you for your contribution! 🚀

**Comment by marcusicaro** [comment]
Thanks! Have a great night/day.

**Pull request merged**